crates/jstzd/src/config.rs

```diff
@@ -69,18 +80,18 @@ pub(crate) async fn build_config(
         &octez_client_config,
     )?;
 
-    // TODO: https://linear.app/tezos/issue/JSTZ-238/deserialize-rollup-config
-    // Dummy rollup config for now
     let octez_node_endpoint = octez_node_config.rpc_endpoint.clone();
     let octez_rollup_config = OctezRollupConfigBuilder::new(
         octez_node_endpoint,
         octez_client_config.base_dir().into(),
         SmartRollupHash::from_base58_check(JSTZ_ROLLUP_ADDRESS).unwrap(),
-        "bootstrap1".to_string(),
-        "dummy-kernel".into(),
+        ROLLUP_OPERATOR_ACCOUNT_ALIAS.to_string(),
+        jstz_rollup_path::kernel_installer_path(),
     )
-    .set_data_dir(RollupDataDir::Temp)
-    .set_rpc_endpoint(&Endpoint::localhost(8000))
+    .set_data_dir(RollupDataDir::TempWithPreImages {
```

Comment:
Note from the offline discussion: this will need to be updated later on so that
* `Config` gets one more member `octez_rollup: Option<OctezRollupConfigBuilder>`
* if `config.octez_rollup` is provided, use that as the config builder and fill in the missing bits, e.g. `octez_node_endpoint`
* if not, create a new builder with a default path, possibly taken from some environment variable so that we can easily control the default path (which will be essential when it comes to packaging and distribution)

Builds with this implementation do not work outside the jstz repo. We're proceeding with this for now just so that we can at least get a concrete (though not distributable) full implementation.